ugvc/pipelines/lpr/train_lib_prep_recalibration_model.py

```diff
@@ -0,0 +1,114 @@
+import os
+
+from simppl.cli import get_parser, get_simple_pipeline
+
+from ugvc.pipelines.mrd import featuremap_to_dataframe
+
+
+def init_parser():
+    parser = get_parser("train_lib_prep_recalibration_model", run.__doc__)
+    parser.add_argument("--out_dir", required=True)
+    parser.add_argument("--ref_fasta", required=True)
+    parser.add_argument("--substitution_analysis_wid", help="workflow-id to take substitution analyss otputs from")
+    parser.add_argument("--featuremap_vcf", help="full featuremap vcf file")
+    parser.add_argument("--notebooks_dir", required=True)
+    parser.add_argument("--calls_vcf", help="variant calling vcf file")
+    parser.add_argument("--tp_min_af", type=float, help="min allele-frequency to consider a variant tp", default=0.9)
+    parser.add_argument("--fp_max_af", type=float, help="max allele-count to consider a variant fp", default=0.04)
+    parser.add_argument("--output_suffix", default="")
+    parser.add_argument("--balance_motifs", default=False, action="store_true")
+    parser.add_argument("--balance_tp_fp", default=False, action="store_true")
+
+    return parser
+
+
+def run(argv):
+    """
+    Lib-prep recalibration model training pipeline
+    """
+    parser = init_parser()
+    sp = get_simple_pipeline(parser, argv=argv, name=__name__)
+    args = parser.parse_args(argv[1:])
+    print(args)
+
+    out_dir = args.out_dir
+    os.makedirs(out_dir, exist_ok=True)
+
+    featuremap_vcf, unfiltered_calls = localize_inputs(args, sp)
+
+    tp_args = [f"{out_dir}/estimated_tps.{suffix}" for suffix in ("bed", "vcf.gz", "parquet")]
+    fp_args = [f"{out_dir}/estimated_fps.{suffix}" for suffix in ("bed", "vcf.gz", "parquet")]
+    tp_args.extend([args.tp_min_af, 1])
+    fp_args.extend([0, args.fp_max_af])
+    afs_bed = f"{out_dir}/afs.bed"
+
+    if args.calls_vcf:
+        calls_vcf = f"{out_dir}/calls.snps.pass.vcf.gz"
+        featuremap_on_calls = f"{out_dir}/featuremap_on_calls.vcf.gz"
+        sp.print_and_run(f"bcftools view --type snps -f PASS {unfiltered_calls} -Oz -o {calls_vcf}")
+        sp.print_and_run(
+            f"bedtools intersect -a {featuremap_vcf} -b {calls_vcf} -header | "
+            f"bcftools view -Oz -o {featuremap_on_calls}"
+        )
+        featuremap_vcf = featuremap_on_calls
+        sp.print_and_run(
+            f"bcftools query -f '%CHROM %POS [%VAF]\\n' {calls_vcf} | grep -v , | "
+            + "awk -v OFS='\t' '{print $1,$2-1,$2,$3}' "
```

Comment:
```suggestion
            bcftools query -f '%CHROM\t%POS0\t%POS\t[%VAF]\n' {calls_vcf}
```
Would do the trick without the awk command